packages/terra-form-select/src/native-select/NativeSelect.module.scss

```
@@ -34,7 +34,9 @@
       background-color: var(--terra-form-select-native-focus-background-color, #fff);
       background-size: var(--terra-form-select-native-focus-background-size, 0 0);
       border-color: var(--terra-form-select-native-focus-border-color, #4cb2e9);
+      outline: var(--terra-form-select-native-focus-keyboard-outline, 2px dashed #000);
```

```suggestion
      outline: var(--terra-form-select-native-focus-mouse-outline, 2px dashed #000);
```
you need to create these variables in other theme files as well

---

Updated.